src/main/java/com/todoist/pojo/Note.java

```diff
@@ -9,17 +10,20 @@
     private long postedUid;
     private Set<Long> uidsToNotify;
     private FileAttachment fileAttachment;
+    private Map<String, long[]> reactions;
     private long projectId;
     private long itemId;
     private boolean archived;
 
     public Note(long id, String content, long posted, long postedUid, Collection<Long> uidsToNotify,
-                FileAttachment fileAttachment, long projectId, long itemId, boolean archived, boolean deleted) {
+                FileAttachment fileAttachment, Map<String, long[]> reactions, long projectId, long itemId,
+                boolean archived, boolean deleted) {
         super(id, deleted);
         this.content = content;
         this.posted = posted;
         this.postedUid = postedUid;
         this.uidsToNotify = Utils.unmodifiableSet(uidsToNotify);
+        this.reactions = Utils.unmodifiableMap(reactions);
```

Comment:
Super minor, but the field/arg order is `fileAttachment` and then `reactions`, so we should initialize them in the same order as well. 🙂